README.md

```diff
@@ -137,25 +139,25 @@ on [their official website](https://docs.anaconda.com/miniconda/).
 
 1. Clone this repository from GitHub:
 ```bash
-$ git clone git@github.com:UBC-MDS/fixml.git
+git clone git@github.com:UBC-MDS/fixml.git
 ```
 
 2. Create a conda environment:
 
 ```bash
-$ conda env create -f environment.yaml
+conda env create -f environment.yaml
 ```
 
 3. Activate the newly created conda environment (default name `fixml`):
 
 ```bash
-$ conda activate fixml
+conda activate fixml
 ```
 
 4. Use `poetry` which is preinstalled in the conda environment to create a local package install:
 
 ```bash
-$ poetry install
+poetry install
 ```
 
```

Comment:
@SoloSynth1 although it's mentioned earlier in the "Important" about setting the API key, i think it's better to explicitly have a instruction for that, in case people (like what Tiffany did last time) just try to copy and paste to follow the instruction.

```suggestion
# 5. add .env with API key attached:
echo "OPENAI_API_KEY=..." > .env
```

Reply:
This will be addressed in upcoming PR for #205.

Reply:
noted